doc/config.example.yaml

```diff
@@ -60,7 +60,7 @@ validator:
   #
   # DO NOT USE THOSE VALUES ANYWHERE
   # Private seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
-  # Public address:  GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ
+  # Public address:  GDCOMMO272NFWHV5TQAIQFEDLQZLBMVVOJTHC3F567ZX4ZSRQQQWGLI3
   seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
```

Comment:
How come the seed didn't change ?

Reply:
I'll revert the change as it is just an example file.

Reply:
Still here

Reply:
Gone now.